manim/mobject/svg/text_mobject.py

```diff
@@ -1056,11 +1058,510 @@ def text2svg(self):
             logger.debug(f"Setting Text {text}")
             pangocairocffi.show_layout(context, layout)
             offset_x += pangocffi.units_to_double(layout.get_size()[0])
         surface.finish()
         return file_name
+
+
+class MarkupText(SVGMobject):
+    r"""Display (non-LaTeX) text rendered using `Pango <https://pango.gnome.org/>`_.
+
+    Text objects behave like a :class:`.VGroup`-like iterable of all characters
+    in the given text. In particular, slicing is possible. Text can be formatted
+    using different tags:
+
+    - ``<b>bold</b>``, ``<i>italic</i>`` and ``<b><i>bold+italic</i></b>``
+    - ``<ul>underline</ul>`` and ``<s>strike through</s>``
+    - ``<tt>typewriter font</tt>``
+    - ``<big>bigger font</big>`` and ``<small>smaller font</small>``
+    - ``<sup>superscript</sup>`` and ``<sub>subscript</sub>``
+    - ``<span underline="double">double underline</span>``
+    - ``<span underline="error">error underline</span>``
+    - ``<span font_family="sans">temporary change of font</span>``
+    - ``<color col="RED">temporary change of color</color>``; colors can be specified as Manim constants like ``RED`` or ``YELLOW``
+    - ``<gradient from="YELLOW" to="RED">temporary gradient</gradient>``; colors specified as above
+
+    If your text contains ligatures, the :class:`MarkupText` class may incorrectly determine
+    the first and last letter to be colored. This is due to the fact that e.g. ``fl``
+    are two characters, but can be set as one single glyph, a ligature. If your language does
+    not depend on ligatures, consider setting ``disable_ligatures=True``. If you cannot
+    or do not want to do without ligatures, the ``gradient`` and ``color`` tag support
+    an optional attribute ``offset`` which can be used to compensate for that error.
+    Usage is as follows:
+
+    - ``<color col="RED" offset="1">red text</color>`` to *start* coloring one letter earlier
+    - ``<color col="RED" offset=",1">red text</color>`` to *end* coloring one letter earlier
+    - ``<color col="RED" offset="2,1">red text</color>`` to *start* coloring two letters earlier and *end* one letter earlier
+
+    Specifying a second offset may be necessary if the text to be colored does
+    itself contain ligatures. The same can happen when using HTML entities for
+    special chars.
+
+    Escaping of special characters: ``>`` *should* be written as ``&gt;`` whereas ``<`` and
+    ``&`` *must* be written as ``&lt;`` and ``&amp;``.
+
+    You can find more information about Pango markup formatting at the
+    corresponding documentation page:
+    `Pango Markup <https://developer.gnome.org/pango/stable/pango-Markup.html>`_.
+    Please be aware that not all features are supported by this class and that
+    the ``<color>`` and ``<gradient>`` tags mentioned above are not supported by Pango.
+
+    Parameters
+    ----------
+    text : :class:`str`
+        The text that need to created as mobject.
```

Comment:
I don't think you need to specify the parameter as it is done through typing you done.
```suggestion
    text
        The text that need to created as mobject.
```